canonical-websites/www.ubuntu.com
# Build the Ubuntu core full disk encryption takeover

## Done
Build the Ubuntu core full disk encryption takeover with animation.

**Missing landing page link**

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to the homepage
- Reload until you see the Full disk encryption takeover
- Check the copy matches [the copy doc](https://docs.google.com/document/d/1r9wlmgwrkFahKHWi-Vf6xE8YqIRSklGWY4o-0w6ytQ4/edit?ts=5c580bd9#heading=h.meo0azuz09li)
- Check the design matches [the visual design](https://github.com/ubuntudesign/web-squad/issues/968)
- Check the animation makes you smile :smile: 

## Issue / Card
Fixes https://github.com/ubuntudesign/web-squad/issues/969
